scripts/build_mirror_config.sh

```diff
@@ -6,7 +6,8 @@ export ARCHITECTURE=$2
 export DISTRIBUTION=$3
 
 DEFAULT_MIRROR_URL_PREFIX=http://packages.trafficmanager.net
-MIRROR_VERSION_FILE=files/build/versions/default/versions-mirror
+MIRROR_VERSION_FILE=
+[[ "$SONIC_VERSION_CONTROL_COMPONENTS" == *deb* || $SONIC_VERSION_CONTROL_COMPONENTS == *all* ]] && MIRROR_VERSION_FILE=files/build/versions/default/versions-mirror
 [ -f target/versions/default/versions-mirror ] && MIRROR_VERSION_FILE=target/versions/default/versions-mirror
```

Comment:
@xumia if target/versions/default/versions-mirror exists, MIRROR_VERSION_FILE will be set to target/versions/default/versions-mirror anyway? It will overwrite the MIRROR_VERSION_FILE set by "MIRROR_VERSION_FILE=
[[ "$SONIC_VERSION_CONTROL_COMPONENTS" == *deb* || $SONIC_VERSION_CONTROL_COMPONENTS == *all* ]] && MIRROR_VERSION_FILE=files/build/versions/default/versions-mirror"

Reply:
Yes, it is as expected, the file files/build/versions/default/versions-mirror will be copied to target folder if not exist.

Reply:
@xumia can I understand before this change, files/build/versions/default/versions-mirror will be used by default. With the change, it would be optional, and target/versions/default/versions-mirror has higher priority than files/build/versions/default/versions-mirror?

Reply:
The file target/versions/default/versions-mirror has higher priority than files/build/versions/default/versions-mirror, the behaviors is not changed.

files/build/versions/default/versions-mirror only used when reproducible enabled, because files/build/versions/ is designed for reproducible build.